packages/angular/cli/src/command-builder/command-module.ts

```diff
@@ -140,8 +140,11 @@ export abstract class CommandModule<T extends {} = {}> implements CommandModuleI
 
     // Gather and report analytics.
     const analytics = await this.getAnalytics();
+    let periodicAnalyticsFlush: (() => Promise<void>) | undefined;
```

Comment:
Nit: Name this variable more clearly after what it will do, like `stopPeriodicFlushes`, so this makes more sense when it is called.